src/Microsoft.QualityGuidelines.Analyzers/Core/MicrosoftQualityGuidelinesAnalyzersResources.resx

```diff
@@ -210,4 +210,13 @@
   <data name="RethrowToPreserveStackDetailsTitle" xml:space="preserve">
     <value>Rethrow to preserve stack details.</value>
   </data>
+  <data name="DoNotRaiseExceptionsInExceptionClausesDescription" xml:space="preserve">
+    <value>When an exception is raised in a finally or fault clause, the new exception hides the active exception. When an exception is raised in a filter clause, the run time silently catches the exception. This makes the original error difficult to detect and debug.</value>
```

Comment:
We only handle `finally` clauses (and, as you say, you can't actually make this mistake for fault clauses and filter clauses in C# and VB). So:

`When an exception is raised in a finally clause, the new exception hides the active exception. This makes the original error difficult to detect and debug.`